### ISSUE TITLE
Перевод на английский

### ISSUE THREAD

**Issue by vstakhov** [lapshinmr/samomes#2]
Я тут подумал, что неплохо бы сделать многоязычную версию приложения с использованием vue-i18n. Есть ли интерес в таком изменении?

**Comment by lapshinmr** [comment]
Да, этот очень хорошее предложение и у меня была такая задача в планах :-)
![Screenshot from 2020-05-09 17-24-21](https://user-images.githubusercontent.com/17162919/81474958-fcb85d00-9219-11ea-9784-363b7a75755d.png)
Спасибо за плагин. Я ознакомлюсь с ним в начале следующей недели.

**Comment by vstakhov** [comment]
Я тут потихоньку пишу переключалку, но вся проблема в том, что я совершенно не знаю vue - он кажется каким-то инопланетным. Если что-то получится вменяемое, то сделаю PR. 
Кстати, вот насчет кальция и магния тоже полезно - у Божкова был расчет реминерализации осмоса.

**Comment by lapshinmr** [comment]
Vue прекрасен :-) Хороший наследник React и Angular.

Какого рода переключалку вы хотите? Может я подскажу? В любом случае взглянул бы с удовольствием на ваши изменения.

Я добавил магний в виде чистого сульфата (обычно продается в аптеках в виде кристалов) и семиводный (обычно продается в садовых магазинах в виде белого порошка и называется магнезией). А так же есть кальций в виде сульфата (гибс). Но я пока не разобрался с его растворимостью в воде. Очень странная растворимость в 2 г на литр при 25 градусах. Часто используют хлорид кальция, я его добавлю сегодня. В общем эти элементы можно включить в состав удобрений так же как и другие реагенты. 

Обычно реминирализатор делают в виде смеси двух порошков и замешивают прямо перед применением. Но тем не менее можно сделать пункт "реминирализатор" или что-то подобное, т.е. без применения воды.

Обычно добавление магния и кальция выглядит как-то так в макро:

![Screenshot from 2020-05-10 09-59-43](https://user-images.githubusercontent.com/17162919/81491974-ff13c900-92a4-11ea-8503-e5158fd13e3a.png)

![Screenshot from 2020-05-10 10-01-48](https://user-images.githubusercontent.com/17162919/81492009-4a2ddc00-92a5-11ea-89ba-0700b7077a24.png)

Еще когда составляется расписание и в удобрении присутсвуют кальций или магний, тогда в итоговой таблице повышения концентраций будет показано и повышение общей жесткости.

**Comment by vstakhov** [comment]
Ну, пока я добился того, что язык в менюшке переключается. Но это напрочь поломало router, и я пока не очень понимаю, как это пофиксить, - сказывается отсутствие опыта работы с vue. Можете посмотреть в моем форке (https://github.com/vstakhov/samomes/commit/c82e698b3c6fd313d36f57cedd13ceda7df53f63 - релевантный коммит).

**Comment by lapshinmr** [comment]
Хорошо. Завтра буду разбираться с плагином и посмотрю форк. Спасибо!

**Comment by vstakhov** [comment]
В общем, я добился перевода менюшки. Оказалось, что в vue нельзя просто так взять и заинжектить в router компоненты без рендеринга parent route. Я нашел похожую проблему у людей и применил один из предложенных там костылей. Теперь, вроде, менюшка работает на двух языках, так что можно сделать WIP PR и потихоньку добавлять перевод для всего остального.

**Comment by lapshinmr** [comment]
Моему удивлению нет предела! Спасибо огромное за помощь, завтра буду разбираться.

**Comment by lapshinmr** [comment]
Работаю над переводом

**Comment by vstakhov** [comment]
Вы это делаете в локальном бранче? А то я мог бы взяться за какую-то страницу или функцию.

**Comment by lapshinmr** [comment]
Нет. Сейчас залита последняя версия. Можете брать любую страницу и переводить. Я последнее вермя больше занимался страницей с динамикой элементов в аквариуме.

**Comment by vstakhov** [comment]
В общем, я начал переводить recipies. Оказалось, что процесс это довольно небыстрый, а времени свободного у меня на выходных очень немного, так что закончу, наверное, не раньше, чем до завтра.
Наткнулся, кстати, на https://rotalabutterfly.com/nutrient-calculator.php - там есть один интересный момент, что предлагается выбрать дозировку согласно какой-то из готовых програм. Хотя это все, конечно, очень сомнительную пользу имеет - EI с виду рай для водорослей (и для производителей удо), а остальное для аквариумов без рыб.

**Comment by lapshinmr** [comment]
Надобности в спешке нет. Когде есть время, тогда и делайте. Спасибо за помощь.

Сейчас работаю над рецептом сухой смеси, чтобы можно было страницу с реминирализацией сделать.

Спасибо за ссылку, ознакомлюсь. Может быть даст какие-то новые мысли или полезности.
